### PR TITLE
Add build time and hostname as Docker labels

### PR DESCRIPTION
**Pull request by bocon13**
Applies to stratum_bf and stratum_bcm containers

### CODE REVIEW

**Comment by bocon13** [comment]
fixes #433

**Comment by bocon13** [comment]
```$ docker inspect --format='{{json .ContainerConfig.Labels}}' stratumproject/stratum-bcm_opennsa:latest | python -m json.tool
{
    "build-machine": "menlo-pdp-lotta-nics",
    "build-timestamp": "20201215-0703UTC",
    "description": "This is the Stratum runtime Docker image for Broadcom switches",
    "maintainer": "Stratum dev <stratum-dev@lists.stratumproject.org>",
    "org.opencontainers.image.revision": "286811fafd0c4f863c1a555f50fd2e4143fe3469-dirty",
    "org.opencontainers.image.source": "git@github.com:stratum/stratum",
    "org.opencontainers.image.version": "286811fafd0c4f863c1a555f50fd2e4143fe3469-dirty",
    "stratum-target": "stratum_bcm_opennsa"
}
```
and
```
$ docker inspect --format='{{json .ContainerConfig.Labels}}' stratumproject/stratum-bf:9.3.0 | python -m json.tool
{
    "bf-sde-version": "9.3.0",
    "build-machine": "menlo-pdp-lotta-nics",
    "build-timestamp": "20201215-0704UTC",
    "description": "This Docker image includes runtime library for Barefoot Tofino switches",
    "maintainer": "Stratum dev <stratum-dev@lists.stratumproject.org>",
    "org.opencontainers.image.revision": "286811fafd0c4f863c1a555f50fd2e4143fe3469-dirty",
    "org.opencontainers.image.source": "git@github.com:stratum/stratum",
    "org.opencontainers.image.version": "286811fafd0c4f863c1a555f50fd2e4143fe3469-dirty",
    "stratum-target": "stratum_bf"
}
```

**Review by pudelkoM — DISMISSED** [review]

File: stratum/hal/bin/barefoot/docker/build-stratum-bf-container.sh [hunk 1]
```diff
@@ -135,6 +135,8 @@ fi
 
 DOCKER_BUILD_OPTS+="--label stratum-target=$STRATUM_TARGET "
 DOCKER_BUILD_OPTS+="--label bf-sde-version=$SDE_VERSION "
+DOCKER_BUILD_OPTS+="--label build-timestamp=$(date +%Y%m%d-%H%M%Z) "
```
Comment (pudelkoM) [inline stratum/hal/bin/barefoot/docker/build-stratum-bf-container.sh:138]: This works on both linux and MacOS:

```
> date '+%F %T%z'
2020-12-15 11:25:47-0800
```

File: stratum/hal/bin/bcm/standalone/docker/build-stratum-bcm-container.sh [hunk 2]
```diff
@@ -79,6 +79,8 @@ if [ "$(docker version -f '{{.Server.Experimental}}')" = "true" ]; then
 fi
 
 DOCKER_BUILD_OPTS+="--label stratum-target=$STRATUM_TARGET "
+DOCKER_BUILD_OPTS+="--label build-timestamp=$(date +%Y%m%d-%H%M%Z) "
```
Comment (bocon13) [inline stratum/hal/bin/bcm/standalone/docker/build-stratum-bcm-container.sh:82]:
```suggestion
DOCKER_BUILD_OPTS+="--label build-timestamp=$(date +%FT%T%z) "
```

**Pull request merged**